bin/p2-launch/main.go

```diff
@@ -36,3 +54,73 @@ func main() {
 		log.Fatalln("Unsuccessful launch of one or more things in the manifest")
 	}
 }
+
+func authorize(manifest pods.Manifest) error {
```

Comment:
Is this copied from `setup.go`? Seems like it should be provided by something in the auth package.